src/routes/Dashboard/DropDown.jsx

```diff
@@ -0,0 +1,30 @@
+import {useEffect, useState} from "react";
+import {Link} from "react-router-dom";
+import classNames from "classnames";
+import onClickOutside from "utils/onClickOutside";
+
+export default function DropDown({setShowProfile, dropDownRef, className}) {
+    const [listening, setListening] = useState(false);
+
+    useEffect(onClickOutside(
```

Comment:
is this working?

You're not setting the ref with `useRef` as in the docs

Reply:
![image](https://user-images.githubusercontent.com/923182/197598252-181ec88f-c05a-43cf-a2d7-c15ab259a990.png)